### PR TITLE
Update synologydsm.markdown

### PR DESCRIPTION
**Pull request by kuchel77**
**Description:**

Update the version of DSM required with this sensor, so that if people have issues with DSM V5, they know why it isn't working.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Is there a parent PR?

File: source/_components/synologydsm.markdown [hunk 1]
```diff
@@ -55,6 +55,11 @@ ssl:
   required: false
   default: true
   type: boolean
+scan_interval:
```
Comment (frenck) [inline source/_components/synologydsm.markdown:58]: This is a platform option, not an option for Synology specific.
Please remove.

File: source/_components/synologydsm.markdown [hunk 2]
```diff
@@ -149,3 +154,9 @@ This sensor will wake up your Synology NAS if it's in hibernation mode.
 Due to the nature of the Synology DSM API it is required to grant the user admin rights. This is related to the fact that utilization information is stored in the core module.
 
 When creating the user it is possible to deny access to all locations and applications. By doing this the user will not be able to login to the web interface or view any of the files on the Synology NAS. It is still able to read the utilization and storage information using the API.
+
+## Synology DSM Versions
+
+Due to a breaking change in the Synology API from DSM version 5 to version 6, some components of this plugin work on with DSM Version 6. This in particular seems to affect the volume and disk settings.
```
Comment (frenck) [inline source/_components/synologydsm.markdown:160]: Maybe this should be added to <https://alerts.home-assistant.io> instead of the documentation?

**Review by frenck — CHANGES_REQUESTED** [review]
@kuchel77 Since you are targeting the `next` branch, I assume there is also a PR on the main codebase? Could you be so kind to provide the link to it?

**Comment by frenck** [comment]
Found the parent PR: home-assistant/home-assistant#27110

Since it has been closed, I'll go ahead and close this PR as well.

**Pull request closed without merging**